package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "detect-changes-action",
-  "version": "1.0.0",
+  "version": "1.0.5",
```

Comment:
We shouldn't change this manually, I'll try to create a PR to update automatically.

Reply:
This might even be based on me testing in my own repo and needing to go through multiple iterations.

Reply:
I understand, I would prefer not to make this change because it can be confusing with the auto-created releases in this repository.